Assets/LeapMotion/Scripts/DetectionUtilities/DetectorLogicGate.cs

```diff
@@ -0,0 +1,79 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using System.Collections.Generic;
+
+namespace Leap.Unity {
+
+  public class DetectorLogicGate : Detector {
+    public List<Detector> Detectors;
+    public bool AddAllDetectorsOnAwake = true;
```

Comment:
Maybe rename to AddSiblingDetectorsOnAwake instead?  It's not immediately obvious what 'All' refers to.